unable to update from kops 1.21 -> 1.22 beta 1

/kind bug

trying to update from kops 1.21 -> kops 1.22 beta 1




```
% ./kops edit cluster --set=spec.kubernetesVersion=1.22.1
% ./kops update cluster
I0831 13:49:33.145584   22809 featureflag.go:169] Unknown FeatureFlag "EnableExternalCloudController"
I0831 13:49:33.180609   22809 s3context.go:92] Found S3_ENDPOINT="https://s3.eu-north-1.amazonaws.com", using as non-AWS S3 backend
W0831 13:49:38.527989   22809 urls.go:71] Using base url from KOPS_BASE_URL env var: "https://elisa-devops-public.s3.eu-north-1.amazonaws.com/kops/1.22.0-beta.1"
I0831 13:49:39.794218   22809 apply_cluster.go:486] Gossip DNS: skipping DNS validation
W0831 13:49:40.293288   22809 model.go:357] overriding log level in manifest (embedded etcd manifest), new level is 1
W0831 13:49:40.293373   22809 model.go:521] overloading ENV var in manifest (embedded etcd manifest) with ETCD_MANAGER_DAILY_BACKUPS_RETENTION=30d
W0831 13:49:40.294062   22809 model.go:357] overriding log level in manifest (embedded etcd manifest), new level is 1
W0831 13:49:40.294121   22809 model.go:521] overloading ENV var in manifest (embedded etcd manifest) with ETCD_MANAGER_DAILY_BACKUPS_RETENTION=30d
panic: semver: Parse(2.12): No Major.Minor.Patch elements found

goroutine 1 [running]:
k8s.io/kops/vendor/github.com/blang/semver/v4.MustParse(0x5896e02, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	vendor/github.com/blang/semver/v4/semver.go:363 +0x245
k8s.io/kops/upup/pkg/fi/cloudup/openstack.useLoadBalancerVIPACL(0x61dbc88, 0xc000248510, 0x0, 0x17, 0xc0006bbd58)
	upup/pkg/fi/cloudup/openstack/cloud.go:693 +0x350
k8s.io/kops/upup/pkg/fi/cloudup/openstack.(*openstackCloud).UseLoadBalancerVIPACL(0xc000248510, 0x61dbc88, 0xc000248510, 0x0)
	upup/pkg/fi/cloudup/openstack/cloud.go:668 +0x6a
k8s.io/kops/pkg/model/openstackmodel.(*OpenstackModelContext).UseVIPACL(0xc000b14180, 0xc0006daed8)
	pkg/model/openstackmodel/context.go:53 +0x4a
k8s.io/kops/pkg/model/openstackmodel.(*FirewallModelBuilder).Build(0xc000f7a060, 0xc000c3dc70, 0x0, 0x0)
	pkg/model/openstackmodel/firewall.go:577 +0xb85
k8s.io/kops/upup/pkg/fi/cloudup.(*Loader).BuildTasks(0xc0006db850, 0xc000a6fc20, 0x7, 0x7, 0xb)
	upup/pkg/fi/cloudup/loader.go:45 +0xbe
k8s.io/kops/upup/pkg/fi/cloudup.(*ApplyClusterCmd).Run(0xc0006dbc38, 0x61540f0, 0xc00012c008, 0x0, 0x0)
	upup/pkg/fi/cloudup/apply_cluster.go:674 +0x1d3c
main.RunUpdateCluster(0x61540f0, 0xc00012c008, 0xc000798948, 0x60fe380, 0xc00000e018, 0xc000519e40, 0xc000985d40, 0x0, 0x0)
	cmd/kops/update_cluster.go:296 +0x6a8
main.NewCmdUpdateCluster.func1(0xc000a4a000, 0x84443b0, 0x0, 0x0, 0x0, 0x0)
	cmd/kops/update_cluster.go:111 +0x5d
k8s.io/kops/vendor/github.com/spf13/cobra.(*Command).execute(0xc000a4a000, 0x84443b0, 0x0, 0x0, 0xc000a4a000, 0x84443b0)
	vendor/github.com/spf13/cobra/command.go:856 +0x472
k8s.io/kops/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x83f4ba0, 0x84443b0, 0x0, 0x0)
	vendor/github.com/spf13/cobra/command.go:974 +0x375
k8s.io/kops/vendor/github.com/spf13/cobra.(*Command).Execute(...)
	vendor/github.com/spf13/cobra/command.go:902
main.Execute()
	cmd/kops/root.go:95 +0x8f
main.main()
	cmd/kops/main.go:20 +0x25
```

--- a/upup/pkg/fi/cloudup/openstack/cloud.go
+++ b/upup/pkg/fi/cloudup/openstack/cloud.go
@@ -690,7 +690,7 @@ func useLoadBalancerVIPACL(c OpenstackCloud) (bool, error) {
 		return false, err
 	}
 	// https://github.com/kubernetes/cloud-provider-openstack/blob/721615aa256bbddbd481cfb4a887c3ab180c5563/pkg/util/openstack/loadbalancer.go#L108
-	return ver.Compare(semver.MustParse("2.12")) > 0, nil
+	return ver.Compare(semver.MustParse("2.12.0")) > 0, nil
 }
 
 type Address struct {

